clients/linux/src/account.rs

```diff
@@ -225,11 +225,13 @@ pub struct Sidebar {
 
 impl Sidebar {
     fn new(m: &Messenger, s: &Settings) -> Self {
+        let scroll = GtkScrolledWindow::new::<Adjustment, Adjustment>(None, None);
```

Comment:
`Adjustment` is already imported as `GtkAdjustment`. Can drop the import above and use:
```suggestion
        let scroll = GtkScrolledWindow::new::<GtkAdjustment, GtkAdjustment>(None, None);
```